code/modules/multiz/zmimic/mimic_movable.dm

```diff
@@ -81,16 +81,15 @@
 	return ..()
 
 /atom/movable/openspace/multiplier/proc/copy_lighting(atom/movable/lighting_overlay/LO, use_shadower_mult = TRUE)
-	appearance = LO
-	layer = MIMICED_LIGHTING_LAYER
-	plane = OPENTURF_MAX_PLANE
-	blend_mode = BLEND_MULTIPLY
-	set_invisibility(INVISIBILITY_NONE)
+	var/mutable_appearance/MA = mutable_appearance(LO)
```

Comment:
Shouldn't this be `new /mutable_appearance()`? Afaict, this looks like it shouldn't work. No point paying the proc overhead for the native behavior anyways though.

Reply:
oh fuck lol you're right